pkg/reconciler/revision/resources/deploy_test.go

```diff
@@ -972,3 +1040,40 @@ func TestMakeDeployment(t *testing.T) {
 		})
 	}
 }
+
+func getSecretInformer() corev1informer.SecretInformer {
+	fake := kubefake.NewSimpleClientset()
+	informer := k8sinformers.NewSharedInformerFactory(fake, 0)
+	return informer.Core().V1().Secrets()
+}
+
+func addSecretToInformer(fake *kubefake.Clientset, secretName string, namespace string) func(si corev1informer.SecretInformer) corev1informer.SecretInformer {
+	return func(si corev1informer.SecretInformer) corev1informer.SecretInformer {
+		ownerObj := &corev1.Service{
```

Comment:
Does this need to be so elaborate with an owner objection and all? Does the code care about that?

Reply:
Yeah this seems excessive. I've removed these.